testsuite/pytests/test_spatial/test_layerNodeCollection.py

```diff
@@ -53,6 +54,59 @@ def test_addTwoLayers(self):
         with self.assertRaises(nest.kernel.NESTError):
             c = layer1 + layer2
 
+    def test_extent_center_mask(self):
+        """Correct extent and center when connecting with mask"""
+        nest.SetKernelStatus({'rng_seed': 1234})
+        # Allowing up to 5 percent relative difference from expected number of connections,
+        # because of randomness.
+        rel_limit = 0.05
```

Comment:
Do you need to allow for 5% variation? I haven't done the maths ...

Reply:
Because nodes are randomly distributed in space, the actual number of connections won't be *exactly* the calculated number of expected connections. With 100 nodes, 5% seemed like a sensible value. Do you have a suggestion for a more suitable value?

Reply:
No, I don't really have a better value. I just wondered about the sensitivity of the test. I think one could work out the standard deviation of the expected number of connections based on Eq (4.8) of [Daniel Hjertholm's master thesis](https://nmbu.brage.unit.no/nmbu-xmlui/handle/11250/189117), but for the sake of time it is probably easier to run the test with, let's say, 100 different seeds and measure the variation. And would the test trigger reliably without this fix in place?

Reply:
@hakonsbm Could you put this information in a comment in the test?

Reply:
@heplesser Done in 41e8c42ad0f06ce4a8e0d8bd7f6f5e8f32b6f17f.